src/calculateRentalCost.js

```diff
@@ -4,7 +4,22 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const price = 40;
+  const shortTerm = 3;
+  const longTerm = 7;
+  const shortTermDisc = 20;
+  const longTermDisc = 50;
+  const noDisc = price * days;
+
+  if (days >= shortTerm && days < longTerm) {
+    return noDisc - shortTermDisc;
+  }
+
+  if (days >= longTerm) {
+    return noDisc - longTermDisc;
+  }
+
+  return noDisc;
```

Comment:
The function `calculateRentalCost` is well-named and descriptive, and the use of constants instead of literals is good practice as it makes the code more readable and maintainable. However, the condition on line 14 can be simplified. Since you are returning from the function if `days` is greater than or equal to `longTerm`, you don't need to check if `days` is less than `longTerm` in the condition on line 14. If the code execution reaches that point, it's already guaranteed that `days` is less than `longTerm`.